src/main/kotlin/com/terraformation/backend/admin/AdminDocumentController.kt

```diff
@@ -0,0 +1,134 @@
+package com.terraformation.backend.admin
```

Comment:
The PR comment made me think the new files were all going to be under `com.terraformation.backend.documentproducer` but this one looks like it's going in the Terraware admin package. Intentional?

Reply:
I will update the description, this is the only class that deviates from the pattern because so far all of our admin controllers reside in the same place.